.github/workflows/ci.yml

```diff
@@ -9,9 +9,9 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
       - name: Setup Scala
-        uses: olafurpg/setup-scala@v10
+        uses: olafurpg/setup-scala@v14
         with:
-          java-version: "adopt@1.11"
+          java-version: "openjdk@1.17"
```

Comment:
This I think is a bit dangerous too. Even if the generated bytecode is compatible with older JDKs, I think there is nothing that would detect uses of APIs not available in Java 11.

Reply:
i did it only because release step uses it, which to me was bit unusual, potentially dangerous, checking CI with 11 version but doing release with 17, or is there specific reason to do that?

Reply:
Ah, if we already use 17 to release then yep, it makes sense to use the same version.